### DISCUSSION TITLE
[Feature] number of photos with a face

### DISCUSSION THREAD

**Discussion by Stooovie** [immich-app/immich#4024] [Feature Request]
### The feature

I'd like a feature that would tell me a number of photos where a certain face is recognized, maybe even with first and last year where photos of this person were found. This would be the perfect place to have it:

![Screenshot 2023-09-08 at 10 38 40](https://github.com/immich-app/immich/assets/10572373/dbfea8bb-2cb1-4375-a79c-071ed0ee1d8a)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by akropp** [comment]
Yes, this would be very helpful.  And perhaps also a count on each face image on the Faces tab.
